typescript/common/script/package.json

```diff
@@ -84,9 +84,9 @@
       "dependencies": [
         "build:jco"
       ],
-      "command": "cat ./wit/*.wit | ./scripts/generate-wit-module.sh > ./lib/index.js",
+      "command": "cat ./wit/*.wit | ../module/scripts/generate-wit-module.sh > ./lib/index.js",
```

Comment:
It's a small thing, but I used these symlinks because maintaining `package.json` is a huge chore and it's nice to just modify one of them and copy it over to the others without modification.

Reply:
hah this is the one that I thought would be most contentious.
I _believe_ git handles symlinks by setting the link up when it writes the file.
so when going between host and virtual FS the symlink intermittently breaks.

as an alternative, I could wrap it in its own package so it can be called with npx?

Reply:
> as an alternative, I could wrap it in its own package so it can be called with npx?

SGTM!

Reply:
- https://github.com/commontoolsinc/system/issues/66

Reply:
- https://github.com/commontoolsinc/system/issues/66